.github/workflows/docs-sync-troubleshooting.yml

```diff
@@ -29,11 +29,17 @@ jobs:
           sparse-checkout: |
             apps/docs
 
-      - name: Set up Node.js
+      - uses: pnpm/action-setup@v4
+        name: Install pnpm
+        with:
+          version: 9.10.0
```

Comment:
You can leave out the version. According to the [docs](https://github.com/pnpm/action-setup?tab=readme-ov-file#version), it will use the `packageManager` key in `package.json`.